### PR TITLE
Part3: Orders

### PR DESCRIPTION
**Pull request by uzairali245**
done

### CODE REVIEW

File: src/Orders.h [hunk 1]
```diff
@@ -16,9 +16,13 @@ class OrdersList
 public:
```
Comment (HAOlleik) [inline src/Orders.h:16]: Remove file, it is repetition of order.h

**Review by HAOlleik — DISMISSED** [review]

**Review by HAOlleik — APPROVED** [review]
LGTM

**Review by Zain-MKhan — APPROVED** [review]
Looks good

**Review by Yason287 — COMMENTED** [review]
Looks good

**Review by daxsis — APPROVED** [review]
didnt read, just for merging

**Pull request merged**